### ISSUE TITLE
pretrained ir models different score every  run

### ISSUE THREAD

**Issue by aydv** [thunlp/OpenMatch#43]
Using  Pretrained IR models for query and doc scoring, at  every run  with  same query and same doc ranking_score are coming different. Tried with  bert, roberta base and large.
 ```
import torch
import OpenMatch as om

query = "Classification treatment COVID-19"
doc = "By retrospectively tracking the dynamic changes of LYM% in death cases and cured cases,
from transformers import AutoTokenizer

tokenizer = AutoTokenizer.from_pretrained("allenai/scibert_scivocab_uncased")
input_ids = tokenizer.encode(query, doc)
model = om.models.Bert("allenai/scibert_scivocab_uncased")
ranking_score, ranking_features = model(torch.tensor(input_ids).unsqueeze(0)) ```

**Comment by Yu-Shi** [comment]
Hello, 

It seems that you are in the pytorch training mode where dropout is enabled automatically. Did you try run it in the evaluation mode? You can set it by `model.eval()`.

Best Regards,

Shi
